engine/crates/runtime-local/src/operation_cache.rs

```diff
@@ -0,0 +1,59 @@
+use runtime::operation_cache::{OperationCache, OperationCacheFactory};
+
+pub struct InMemoryOperationCacheConfig {
+    pub limit: usize,
+}
+
+pub struct InMemoryOperationCacheFactory {
+    pub config: InMemoryOperationCacheConfig,
+}
+
+impl InMemoryOperationCacheFactory {
+    pub fn inactive() -> Self {
+        InMemoryOperationCacheFactory {
+            config: InMemoryOperationCacheConfig { limit: 0 },
+        }
+    }
+}
+
+impl Default for InMemoryOperationCacheFactory {
+    fn default() -> Self {
+        InMemoryOperationCacheFactory {
+            config: InMemoryOperationCacheConfig { limit: 1000 },
+        }
+    }
+}
+
+impl OperationCacheFactory for InMemoryOperationCacheFactory {
+    type Cache<V> = InMemoryOperationCache<V>
```

Comment:
:bug: **Bug**
There is a missing semicolon at the end of the type alias declaration for `Cache<V>` in the `OperationCacheFactory` implementation. This will cause a syntax error and prevent the code from compiling.

```suggestion
    type Cache<V> = InMemoryOperationCache<V>;
    where
```